lib/improver/generate_ancillaries/generate_ancillary.py

```diff
@@ -36,7 +36,8 @@
 from cf_units import Unit
```

Comment:
Pylint is complaining about the import order. I don't claim to understand it's logic but it wan't iris after glob I think. I know you haven't introduced this but we could tidy it up anyway.

Reply:
Now fixed :)